policies/voting-procedure.md

```diff
@@ -0,0 +1,73 @@
+The Public Voting Procedure of OMC
+==================================
+
+The following regulations complement the [OMC Voting Procedures] stated
+in the project bylaws. This policy affects only public votes.
+
+[OMC Voting Procedures]: https://www.openssl.org/policies/omc-bylaws.html#omc-voting
+
+Vote Proposal
+-------------
+
+The votes are proposed in pull requests and issues of the general-policies
+repository on GitHub OpenSSL project.
+
+The vote regarding a policy change proposal is recorded directly in the
+pull request of the policy change proposal.
+
+Any other votes are recorded as separate issues in the repository.
+
+Casting the Votes
+-----------------
+
+The individual members of OMC cast their votes directly in the issue or
+pull request where the vote was proposed.
+
+When the votes are cast during an OMC meeting, the person responsible for
+taking the meeting minutes or the proposer of the vote copies the votes
+cast during the meeting into the issue.
+
+The Final Vote Records
+----------------------
+
+After the vote is closed the proposer of the vote records the final outcome
+and individual member votes in a separate file in the votes subdirectory of
+the repository. The commit with the vote record is pushed directly into
+the master branch of the repository. No pull request is needed.
+
+The file is formatted as follows:
+
+```
+Topic: .
+Proposed by: .
+Issue link: https://github.com/openssl/general-policies/issues/...
+Public: yes
+Opened: yyyy-mm-dd
+Closed: yyyy-mm-dd
+Accepted:  yes/no  (for: X, against: Y, abstained: Z, not voted: T)
+
+  OMC Member A  [  ]
+  OMC Member B  [  ]
+  ...
+```
+
+The individual member votes are recorded as `[+1]` a vote in favour, `[-1]`
+a vote against, `[+0]` an abstention with an inclination in favour,
+`[ 0]` a neutral abstention, `[-0]` an abstention with an inclination
+against, and `[  ]` meaning not voted.
+
+The vote files are named `yyyymmdd-vote-short-id.txt` where the `yyyymmdd`
```

Comment:
So, in technical policies, I see files called `vote-yyyymmdd-vote-short-id.txt`...  are we going to allow those too?  Might as well be consistent in our inconsistency...

Reply:
I've change the filename format in this policy to say `vote-yyyymmdd-vote-short-id.txt` as per current OTC practice. I've also proposed an equivalent change to the OTC policy:
https://github.com/openssl/technical-policies/pull/17